src/api/handlers/guild.ts

```diff
@@ -587,6 +587,36 @@ export async function editEmbed(
   );
 }
 
+/** Returns the widget for the guild. */
+export async function getWidget(guildID: string, options?: { force: boolean }) {
+  const guild = await cacheHandlers.get("guilds", guildID);
+  if (!options?.force) {
+    if (!guild) throw new Error(Errors.GUILD_NOT_FOUND);
+    if (!guild?.widgetEnabled) throw new Error(Errors.GUILD_WIDGET_NOT_ENABLED);
+  }
+
+  return RequestManager.get(`${endpoints.GUILD_WIDGET(guildID)}.json`);
+}
+
+/** Returns the widget image URL for the guild. */
+export async function getWidgetImageUrl(
+  guildID: string,
+  options?: {
+    style?: "shield" | "banner1" | "banner2" | "banner3" | "banner4";
+    force?: boolean;
+  },
+) {
+  const guild = await cacheHandlers.get("guilds", guildID);
+  if (!options?.force) {
+    if (!guild) throw new Error(Errors.GUILD_NOT_FOUND);
+    if (!guild?.widgetEnabled) throw new Error(Errors.GUILD_WIDGET_NOT_ENABLED);
+  }
+
+  return `${endpoints.GUILD_WIDGET(guildID)}.png?style=${
```

Comment:
style=${options?.style || "shield"}

Also are `format` options supported? What happens if u test this with .jpg instead of png

Reply:
No formatting options are not supported.

Reply:
Why wasthis resolved? the change should stillbe applied

Reply:
@itohatweb

Reply:
idk what you mean?

Reply:
The only options is png. Thats the endpoint you cannot change anything.